packages/components/src/Autocomplete/Autocomplete.module.css

```diff
@@ -28,11 +27,12 @@
 }
 
 .options.visible {
-  display: block;
+  position: absolute;
```

Comment:
Changed the popper visibility to use visibility to match the recommendations in the docs
https://popper.js.org/docs/v2/modifiers/hide/
It also helped with an issue when clicking on customRendered options that didn't use the `onMouseDown` callback